selenium_tests/course_info/page_objects/course_info_detail_page_object.py

```diff
@@ -77,8 +82,20 @@ def submit_was_successful(self):
         return True
 
     def _wait_for_input_field_to_be_visible(self, field_name):
-        # only want to access input elements once the values are loaded from the
-        # server (which in this case means the spinner disappears and the input
-        # element becomes visible)
-        WebDriverWait(self._driver, 10).until(lambda s: s.find_element(
-            *Locators.INPUT_BY_FIELD_NAME(field_name)).is_displayed())
+        self._wait_for_angular()
+        # element_to_be_clickable works better with Angular than is_displayed()
```

Comment:
Strange -- this is not showing up in our slack `#github` channel... just noticed it because of the PR-level (vs file-level) comment that did show up in slack.